src/app/modules/main/activity_center/controller.nim

```diff
@@ -140,3 +140,21 @@ proc getMessageById*(self: Controller, chatId, messageId: string): MessageDto =
   if(err.len > 0):
     return MessageDto()
   return message
+
+proc setActiveNotificationGroup*(self: Controller, group: int) =
+  self.activityCenterService.setActiveNotificationGroup(ActivityCenterGroup(group))
+  self.activityCenterService.resetCursor()
+  let activityCenterNotifications = self.activityCenterService.getActivityCenterNotifications()
```

Comment:
Each time we change group, we have to refetch from the DB if I understand correctly. Wouldn't that be slow on apps with a lot of notifications?

Reply:
We fetch 20 just notifications initially, so it should be fast. And it fixes one major bug: now we fetching 20 notifications what match filtering conditions. With ui-side sorting we was needed to fetch **al**l of notifications to be sure we don't miss anything